httpbin/handlers_test.go

```diff
@@ -2190,6 +2190,36 @@ func TestUUID(t *testing.T) {
 	}
 }
 
+func TestValidBase64(t *testing.T) {
+	encodedText := "SFRUUEJJTkdPIGlzIGF3ZXNvbWU="
+	urlStr := "/base64/" + encodedText
+	r, _ := http.NewRequest("GET", urlStr, nil)
+	w := httptest.NewRecorder()
+	handler.ServeHTTP(w, r)
+
+	assertStatusCode(t, w, http.StatusOK)
+	expectedContent := "HTTPBINGO is awesome"
+	gotContent := w.Body.String()
+
+	if gotContent != expectedContent {
+		t.Fatalf("Expected [%s] - Got [%s]", expectedContent, w.Body.String())
+	}
+}
+
+func TestInvalidBase64(t *testing.T) {
```

Comment:
One thing I'd like to move towards in these tests, where it's not too much of a burden, is defining a single top-level `TestBase64` function for an endpoint and defining individual cases within it using `t.Run()`.

Reply:
Ok - will check and update in a few days.

Reply:
Done.